src/main/java/spring/core/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package spring.core.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import spring.core.dto.UserResponseDto;
+import spring.core.model.User;
+import spring.core.service.UserMapper;
+import spring.core.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    private UserMapper userMapper;
+    private UserService userService;
+
+    @Autowired
+    public UserController(UserMapper userMapper, UserService userService) {
+        this.userMapper = userMapper;
+        this.userService = userService;
+    }
+
+    @RequestMapping(value = "/user/inject", method = RequestMethod.PUT)
```

Comment:
Why you use put instead get?

Reply:
Changed to type request GET. Thanx for explanation. =)